modules/tinymce/src/core/test/ts/browser/lists/ApplyDlTest.ts

```diff
@@ -3,9 +3,8 @@ import { LegacyUnit, TinyAssertions, TinyHooks } from '@ephox/wrap-mcagar';
 import { assert } from 'chai';
 
 import Editor from 'tinymce/core/api/Editor';
-import Plugin from 'tinymce/plugins/lists/Plugin';
 
-describe('browser.tinymce.plugins.lists.ApplyDlTest', () => {
+describe('browser.tinymce.core.lists.ApplyDlTest', () => {
   const hook = TinyHooks.bddSetupLight<Editor>({
     plugins: 'lists',
```

Comment:
Since you are keeping this it will try to load the lists plugin from the base URL if it's not already loaded. Not sure what this does for tests though will it be weird. We might end up with a situation where it wouldn't load the webpack version of the plugin in tests when do you development.